lnpeer/peer.go

```diff
@@ -47,7 +47,7 @@ type Peer interface {
 
 	// WipeChannel removes the channel uniquely identified by its channel
 	// point from all indexes associated with the peer.
-	WipeChannel(*wire.OutPoint)
+	WipeChannel(*wire.OutPoint) error
```

Comment:
I don't think we should introduce an error return here.

Reply:
Plus if you do this then you need to fix all the mocks. I don't think it's worth it. See my other comment about garbage collection strategy.